sugarscape.py

```diff
@@ -136,10 +136,11 @@ def configureAgents(self, numAgents):
                 a.decisionModelLookaheadFactor = 0.5
             
             if self.environmentTribePerQuadrant == True:
-                tags = self.generateAgentTags(quadrantIndex)
+                tags = self.generatePerQuadrantAgentTags(quadrantIndex)
                 a.tags = tags
                 a.tribe = a.findTribe()
             randomCell.agent = a
+            a.cell = randomCell
```

Comment:
This should not be necessary since agents are assigned their cells in `Agent` init. Is there a bug we're covering up by fixing it like this?

Reply:
Correct. There was a bug where the number of agents in the interface was less than the number of starting agents. Adding this line seemed to make that go away, but you're right in that it should have no effect. Feel free to cut the line out. I'll see if I can reproduce the issue in a subsequent review.

Reply:
@nkremerh Were you able to reproduce the starting agents issue?

Reply:
No. It may have been introduced elsewhere in my initial changes, but it has stayed gone.